packages/next/src/next-devtools/dev-overlay/components/overview/segment-boundary-trigger.tsx

```diff
@@ -326,6 +252,10 @@ export const styles = `
     min-width: 120px;
   }
 
+  .segment-boundary-dropdown-positioner {
+    z-index: 2147483648;
```

Comment:
temporary solution: just let the z-index matching the segment explorer on indicator dock. It requires more refactoring to move the dock outside of the indicator div.